reference/toolstacks/nodejs/README.md

```diff
@@ -20,59 +20,70 @@ To use Platform.sh and Node.js together, configure the ``.platform.app.yaml``
 file with a few key settings, as described here (a complete example is included
 at the end).
 
-1. Specify the `type` as `nodejs` you should definitely specify a version for example `type: nodejs:4.2`
+1. Specify the `type` as `nodejs`. You may also specify a version for example `type: nodejs:4.4`.
```

Comment:
1) Let's change this to "it's is recommended that you also specify".  Unspecified versions are a time bomb waiting to happen as soon as we ever want to change the default, and/or drop old versions.  I'd almost go as far as documenting it as though it were a requirement.

2) Let's use 6.2 as the example version, as that's the newest version.